### PR TITLE
Fixed bug with --no-merge with directories

### PR DESCRIPTION
**Pull request by ModST**
had a bug with --out and directories with the new flag.

### CODE REVIEW

File: lib/parallel_split_test/command_line.rb [hunk 1]
```diff
@@ -41,7 +41,9 @@ def run(err, out)
 
     # modify + reparse args to unify output
     def modify_out_file_in_args(process_number)
-      @args[out_file_position] = process_number.to_s << "-" << out_file
+      target = out_file.split(".")
+      target[target.length - 2] = target[target.length - 2] << "." << process_number.to_s
```
Comment (grosser) [inline lib/parallel_split_test/command_line.rb]: can you wrap this logic into a method and give it a name/some docs ... maybe File.extname() or so helps ...

Comment (ModST) [inline lib/parallel_split_test/command_line.rb]: I didn't know there was a way better way to do that. Thanks!

**Comment by grosser** [comment]
that looks great, thx!

**Comment by grosser** [comment]
0.6.0

**Pull request merged**